README.md

```diff
@@ -23,30 +24,62 @@ The protocol is defined [here](https://github.com/multiformats/multistream-selec
 
 ## Install
 
+`go-multistream` is a standard Go module which can be installed with:
+
 ```sh
 go get github.com/multiformats/go-multistream
 ```
 
+Note that `go-multistream` is packaged with Gx, so it is recommended to use Gx to install and use it (see Usage section).
+
+
 ## Usage
 
+### Using Gx and Gx-go
+
+This module is packaged with [Gx](https://github.com/whyrusleeping/gx). In order to use it in your own project do:
+
+```sh
+go get -u github.com/whyrusleeping/gx
+go get -u github.com/whyrusleeping/gx-go
+cd <your-project-repository>
+gx init
+gx import https://github.com/multiformats/go-multistream
```

Comment:
it doesn't work with https://

and I didn't even know about this function, nice.